quinn-proto/src/connection.rs

```diff
@@ -41,8 +41,7 @@ pub struct Connection {
     rx_packet: u64,
     /// Time at which the above was received
     rx_packet_time: u64,
-    crypto: Option<Crypto>,
-    prev_crypto: Option<(u64, Crypto)>,
+    cryptos: VecDeque<(CryptoLevel, u64, Crypto)>,
```

Comment:
Code referencing this would be more readable if its element type was a struct with helpfully-named fields.

Reply:
Yeah, totally fair -- I was thinking about it as I was writing it. Fixed this. Let me know if you think of a better name than `CryptoSpace`.